doc/modules/preprocessing.rst

```diff
@@ -474,11 +474,11 @@ then the shape of the transformed input is
     Imputer(add_indicator_features=True, axis=0, copy=True, missing_values='NaN',
         strategy='mean', verbose=0)
     >>> print(imp_with_in.transform(X))                           # doctest: +ELLIPSIS
-    [[ 4.          2.          1.          0.        ]
-     [ 6.          3.66666667  0.          1.        ]
+    [[ 4.          2.          0.          0.        ]
```

Comment:
Thank you, will definitely do that. 

The idea of user specifying seems good but would it always be the case that the user is aware of the features with missing data in the whole training data ?. And even if they do, will it be consistent for all the samples ?

Also if we proceed to add indicator features for every feature irrespective of imputation, something like variance threshold can remove such indicators features that correspond to features with no missing values right ? Or maybe give an option specifically for the indicator features to have all features or just imputed features ?